tests/test_archiver.py

```diff
@@ -21,7 +21,7 @@ def test_scan_archives_all(self):
         self.archiver.scan_archives()
         archives_for_channel = self.archiver.archives_for_channel
         data = self.archiver.get('XF:23IDA-VA:0{DP:1-IP:1}P-I', '2013-08-11', '2013-08-12')
-        print data
+        print(data)
```

Comment:
@tacaswell print is a function in py27. This breaks py26 without the print_function import though.